CSS/contact.css

```diff
@@ -0,0 +1,30 @@
+.contact_form {
```

Comment:
since we will be using less, no need to have the css files as well
you can remove them